services/src/main/java/uk/gov/moj/sdt/services/BulkSubmissionService.java

```diff
@@ -196,7 +204,7 @@ private void enrich(final IBulkSubmission bulkSubmission) {
 
         // Get the Raw XML from the ThreadLocal and insert in the BulkSubmission
         String rawInXml = SdtContext.getContext().getRawInXml();
-        bulkSubmission.setPayload(rawInXml == null ? "".getBytes() : rawInXml.getBytes());
+        bulkSubmission.setPayload(rawInXml.getBytes(StandardCharsets.UTF_8));
```

Comment:
Do we want to re-introduce the null check here?

Reply:
done